.github/workflows/ci.yaml

```diff
@@ -40,4 +40,19 @@ jobs:
           channel: 1.23/stable
           bootstrap-options: "--agent-version 2.9.29"
       - name: Run integration tests
-        run: tox -e integration
+        run: tox -e integration-ha
+  integration-legacy-mysql-test-microk8s:
```

Comment:
nit: integration-legacy-relations sounds better, I am not sure if `-mysql-test-microk8s` adds useful info